src/main/java/org/cdc/gov/sdp/queue/DatabaseQueueProducer.java

```diff
@@ -53,35 +64,45 @@ public void process(final Exchange exchange) throws Exception {
 				queueConnection = queueDataSource.getConnection();
 			}
 
+			if (!tableCreated) {
+				PreparedStatement makeDbIfNeeded = queueConnection.prepareStatement(this.queueTableCreateCommand);
+				String thing = makeDbIfNeeded.toString();
+				System.out.println(thing);
```

Comment:
I'm guilty of this but we should avoid sout.println  and use a logger in its place.  If this isnt needed then it should be removed